Lib/test/test_cmd_line_script.py

```diff
@@ -438,6 +438,15 @@ def test_issue20884(self):
             self.assertEqual(b"", out)
             self.assertEqual(b"", err)
 
+    def test_issue32381(self):
```

Comment:
Instead of `test_issue32381`, I would recommend renaming the method to something more descriptive of the tests, such as `test_windows_ascii_path`.